config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/jinzhu/configor"
+)
+
+//FromJSON will load the requested JSON config into the given config interface
```

Comment:
Let's bail on all package/function level comments unless it is absolutely needed.  I believe I've removed those from the code quality reporting in the Makefile.